pkg/volume/local/local.go

```diff
@@ -128,7 +128,7 @@ func (plugin *localVolumePlugin) NewMounter(spec *volume.Spec, pod *v1.Pod, _ vo
 			mounter:         plugin.host.GetMounter(plugin.GetPluginName()),
 			plugin:          plugin,
 			globalPath:      globalLocalPath,
-			MetricsProvider: volume.NewMetricsStatFS(volumeSource.Path),
+			MetricsProvider: volume.NewMetricsStatFS(plugin.host.GetPodVolumeDir(pod.UID, stringsutil.EscapeQualifiedNameForDisk(localVolumePluginName), spec.Name())),
```

Comment:
can this call `GetPath` instead?

Reply:
`GetPath` is the function of `localVolume` struct, and here, it is the place to construct the `localVolume` struct, so maybe not

Reply:
We can create a private function here which can be called both in `GetPath` and here like other plugins do, but seems not that necessary

Reply:
this is fine then. we just have to make sure if this ever changes in the future we update both places

Reply:
yes, i can send a separate PR for all plugins to maintain the consistent behavior after this release period.